src/PerfView/PerfView.csproj

```diff
@@ -8,6 +8,7 @@
     <StartupObject>PerfView.App</StartupObject>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <ExtrasEnableWpfProjectSetup>true</ExtrasEnableWpfProjectSetup>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
```

Comment:
We probably shouldn't add this. The custom assembly resolver knows how to load AnyCPU assemblies from SupportFileDir, so there's no need to copy them into the amd64 and x86 directories. In other words, you want this:

https://github.com/microsoft/perfview/blob/65a4b1b706fea223e22a470813f2b94766a99762/src/PerfView/Utilities/SupportFiles.cs#L83-L87

Instead of what you are currently relying on:

https://github.com/microsoft/perfview/blob/65a4b1b706fea223e22a470813f2b94766a99762/src/PerfView/Utilities/SupportFiles.cs#L89-L94

Reply:
This is the PerfView resolver, but we're talking about HeapDump.exe here.  That said, your point is valid.  I have created a HeapDump directory for IL dependencies and updated the resolver for HeapDump.exe to look there.